plugins.txt

```diff
@@ -54,7 +54,8 @@ javadoc:217.v905b_86277a_2a_
 jira:3.7.1
 job-dsl:1.79
 junit:1119.va_a_5e9068da_d7
-kubernetes:3636.v84b_a_1dea_6240
+# TODO https://github.com/jenkinsci/kubernetes-plugin/pull/1190
+kubernetes:incrementals;org.csanchez.jenkins.plugins;3650.ve6f9e42d64de
```

Comment:
Note that this is not _currently_ used by the `buildPlugin` patch, for reasons discussed there.

Reply:
…but _is_ used by https://github.com/jenkinsci/bom/pull/1249.